headers/functions/arm9/itcm.h

```diff
@@ -26,6 +27,7 @@ int GetKeyM2N(enum monster_id monster_id);
 int GetKeyM2NBaseForm(enum monster_id monster_id);
 void HardwareInterrupt(void);
 void ReturnFromInterrupt(void);
+void InitDmaTransfer_Standard(int channel, int* src, int* dst, uint16_t word_count);
```

Comment:
Should the pointers be `void*`? DMA seems more generic than specifically integers